src/js/typeahead/helpers.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 module.exports = {
-  sorter: function(input) {
-    return function sorter(a, b) {
+  sorter: function sorter(input) {
+    return function sort(a, b) {
```

Comment:
Do you mean transpiling to ES6 to ES5?
No we are not, and frankly, I don't see the point, especially not for the change you've highlighted above.